src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,38 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, activeTabId, onTabSelected }) => {
+  const selectedTab = tabs.find(tab => activeTabId === tab.id) || tabs[0];
+  const { id, content } = selectedTab;
+  const handleOnClick = tab =>
+    selectedTab.id !== tab.id && onTabSelected(tab.id);
```

Comment:
Better use this approach because it improves readability

```
  const handleOnClick = tab => {
    if (id !== tab.id) {
      onTabSelected(tab.id);
    }
  };
```